build-support/bin/cherry_pick.sh

```diff
@@ -34,6 +34,7 @@ if [[ -z $TARGET_MILESTONE ]]; then
   fi
 fi
 
+# NB: Find all milestones >= $TARGET_MILESTONE
```

Comment:
Thanks! Can you clarify (if not in a code comment then at least here) how this is achieved? That awk cmd is mysterious.

Reply:
Hahaha sure thing. Also it was generated by AI, it should be mysterious.

Reply:
should go back to chatgpt then (just assuming here) and ask it to add the explaining comment. 😂